testament/categories.nim

```diff
@@ -688,8 +689,22 @@ proc runJoinedTest(r: var TResults, cat: Category, testsDir: string) =
 
 # ---------------------------------------------------------------------------
 
-proc processCategory(r: var TResults, cat: Category, options, testsDir: string,
+proc loadSkipFrom(name: string): seq[string] =
+  # One skip per line, comments start with #
+  # used by `nlvm` (at least)
+  try:
+    for line in lines(name):
+      let sline = line.strip()
+      if sline.len > 0 and not sline.startsWith("#"):
+        result.add sline
+  except:
+    echo "Could not load " & name & ", ignoring"
```

Comment:
This now prints "Could not load , ignoring" every time, because the default for `skipFrom` is ``""``.

Reply:
this is fixed in https://github.com/nim-lang/Nim/pull/11168

Reply:
well I had the problem just when executing megatest